tests/foreman/api/test_user.py

```diff
@@ -174,6 +174,23 @@ def test_positive_update_username(self):
                 user = user.update(['login'])
                 self.assertEqual(user.login, login)
 
+    @tier1
+    def test_negative_update_username(self):
+        """Update a user and provide new login.
+
+        :id: 9eefcba6-66a3-41bf-87ba-3e032aee1db2
+
+        :expectedresults: The user's ``login`` attribute is updated.
+
+        :CaseImportance: Critical
+        """
+        user = entities.User().create()
+        for login in invalid_usernames_list():
+            with self.subTest(login):
```

Comment:
since we are transitioning away from unittest, can you convert this subtest to use pytest.parameterize instead?

Reply:
Just so we get it straight: first PR rots for 3 weeks, and now we are going to block it because it does not implement best practice that automation team never announced to entire team?

I don't think that's OK. If this is your only concern, just merge it, please.

Reply:
3 weeks is an exaggeration. Between the last force push and the next comment, was 7 days.
Also, just because a PR has been in progress for a while doesn't mean that suggestions shouldn't e considered.